includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -120,4 +138,50 @@ public function capture_terminal_payment( $request ) {
 			return new WP_Error( 'wcpay_server_error', __( 'Unexpected server error', 'woocommerce-payments' ), [ 'status' => 500 ] );
 		}
 	}
+
+	/**
+	 * Returns customer id from order. Create or update customer if needed.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 */
+	public function create_customer( $request ) {
+		try {
+			$order_id = $request['order_id'];
```

Comment:
Nit: `wc_get_order` accepts both Post object and Post ID. Here we're going to use Post ID.

What's about if we cast this to `int` or even `absint`?

```suggestion
			$order_id = absint( $request['order_id'] );
```

Reply:
The underlying codebase is not requiring strict typing, so no need to change this.